docs/dev_install.rst

```diff
@@ -111,6 +114,17 @@ If you don't have CUDA, go to `this website <https://developer.nvidia.com/cuda-d
 Setting up Taichi for development
 ---------------------------------
 
+- Add the following script to your ``~/.bashrc``:
+
+  .. code-block:: bash
+
+    export TAICHI_REPO_DIR=/home/XXX/taichi  # Path to your taichi repository
+    export PYTHONPATH=$TAICHI_REPO_DIR/python:$PYTHONPATH
+    export PATH=$TAICHI_REPO_DIR/bin:$PATH
+    # export PATH=/opt/llvm/bin:$PATH # Uncomment if your llvm-8 or clang-8 is in /opt
+
+- Execute ``source ~/.bashrc`` to reload shell config.
+
```

Comment:
Could this be confusing given `/home/XXX/taichi` doesn't exist yet?

Reply:
I realized we didn't really mention setting these variables on Windows. Maybe we should mention it here?